docs/src/docs/asciidoc/index.adoc

```diff
@@ -521,8 +521,8 @@ detailed information on configuration options for Hazelcast.
 
 Sessions will be stored in a distributed `Map` in Hazelcast using a <<api-mapsessionrepository,MapSessionRepository>>.
 The `Map` interface methods will be used to `get()` and `put()` Sessions.
-The expiration of a session in the `Map` is handled by Hazelcast's native support for configuring a map's `max-idle-seconds` setting.
-Entries (sessions) that have been idle longer than the max idle setting will be automatically removed from the `Map`.
+The expiration of a session in the `Map` is handled by Hazelcast's support for setting the time to live on an entry when it is `put()` into the `Map`.
```

Comment:
I realized when reading this sentence we have an issue. The `put` method on `IMap` does indeed set the TTL, but what we actually want is the TTI (time to idle) set.